deps/build.jl

```diff
@@ -70,4 +70,8 @@ provides(BuildProcess, (@build_steps begin
     end
 end), [libfftw, libfftwf])
 
-BinDeps.@install Dict([:libfftw => :libfftw, :libfftwf => :libfftwf])
+if is_windows()
+    BinDeps.@install Dict([:libfftw3 => :libfftw, :libfftw3f => :libfftwf])
```

Comment:
@tkelman that's what I thought as well, but look at what we do [here](https://github.com/JuliaLang/BinDeps.jl/blob/cd6e61aac5eea8c73433d3de39c8f84517e3be99/src/dependencies.jl#L995).  We use the `libkey` (which is the key we iterate over the mapping we pass to `BinDeps.@install`) to probe into `load_cache`, and `load_cache` [is initialized](https://github.com/JuliaLang/BinDeps.jl/blob/cd6e61aac5eea8c73433d3de39c8f84517e3be99/src/dependencies.jl#L966-L967) from `d.name`, where `d` is a `LibraryDependency` object.  So the keys of this dictionary need to match the names we pass in to `library_dependency()`